modules/ROOT/pages/6.1-release-notes.adoc

```diff
@@ -0,0 +1,1121 @@
+= TinyMCE 6.1
+:navtitle: TinyMCE 6.1
+:description: Release notes for TinyMCE 6.1
+:keywords: releasenotes, new, changes, bugfixes
+
+include::partial$misc/admon-releasenotes-for-stable.adoc[]
+
+[[overview]]
+== Overview
+
+{productname} 6.1 was released for {enterpriseversion} and {cloudname} on Wednesday, July 13^th^, 2022. It includes {productname} 6.1 and additional changes to premium plugins. These release notes provide an overview of the changes for {productname} 6.1:
+
+* xref:accompanying-premium-plugin-changes[Accompanying Premium Plugin changes]
+* xref:accompanying-premium-skins-and-icon-packs-changes[Accompanying Premium Skins and Icon Packs changes]
+* xref:bug-fixes[Bug fixes]
+* xref:changes[Changes]
+* xref:improvements[Improvements]
+* xref:additions[Additions]
+* xref:known-issues[Known issues]
+
+
+[[accompanying-premium-plugin-changes]]
+== Accompanying Premium Plugin changes
+
+The following premium plugin updates were released alongside {productname} 6.1.
+
+=== A11ychecker 3.0.2
+
+The {productname} vnumtxt release includes an accompanying release of the **A11ychecker** premium plugin.
+
+**A11ychecker** 3.0.2 fixes the following UI bug:
+
+The *Tools > Accessibility Checker* command opens a dialog. As such, its menu title should be *Accessibility Checker…*.
+
+The trailing ellipsis indicates the command will not execute without further user input and is required for all menu commands that present dialogs.
+
+This update corrects both the menu title and the equivalently mis-presented toolbar tool tip text. Both strings now include the required ellipsis and present as *Accessibility Checker…*, as expected.
+
+For information on the **A11ychecker** plugin, see: xref:a11ychecker.adoc[A11ychecker plugin].
+
+
+=== Comments 3.1.0
+
+The {productname} vnumtxt release includes an accompanying release of the **Comments** premium plugin.
+
+**Comments** 3.1.0 introduces the following enhancements:
+
+
+==== A new event log API
+
+A new API, `tinymce.activeEditor.plugins.tinycomments.getEventLog()` is now available.
+
+As its name suggests, it returns a log of all Comment-related events, including:
+
+* when a new comment is added (and who added it).
+
+* when a coment is edited (and who edited it).
+
+* when a reply to a comment is added (and who added the reply).
+
+* when a comment is resolved (and who resolved the comment).
+
+* when a comment is deleted (and who deleted the comment).
+
+The new API also supports an `after` option.
+
+[source,js]
```

Comment:
should be `javascript`

Reply:
Frustratingly, Antora’s [highlight.bundle.js](https://gitlab.com/antora/antora-ui-default/-/blob/HEAD/src/js/vendor/highlight.bundle.js) file won’t load as I type.

But a quick search across the docs source came up with 761 `[source,js]` and no `[source,javascript]` which I’m taking as evidence `js` is the correct string.

I see you resolved this before I got to it, so I’m guessing you came to the same conclusion but I thought I should at least add a quick comment so as to not leave the impression I was ignoring or overlooking feedback.